testkit/build.gradle.kts

```diff
@@ -66,7 +66,7 @@ publishing {
 tasks.withType<Sign>().configureEach {
   onlyIf {
     val isNotSnapshot = !version.toString().endsWith("SNAPSHOT")
-    isNotSnapshot //&& gradle.taskGraph.hasTask(publishToMavenCentral.get())
+    isNotSnapshot && (findProperty("signing.keyId") != null)
```

Comment:
This is so that I could call `./gradlew publishToMavenLocal` without having to sign artifacts but I can remove it if needed.

Reply:
My inclination is that all non-snapshot builds should be signed. Another way to resolve your problem would have been to change the version [here](https://github.com/autonomousapps/dependency-analysis-android-gradle-plugin/blob/main/testkit/build.gradle.kts#L15) to have the `-SNAPSHOT` suffix.

I'll think on this a bit. There's probably something I can do with the script configuration to make it easier for others to contribute (and it may or may not be your proposal here).

Reply:
I considered changing the version but I don't like having local changes, I always end up comiting them in source control...
Another option is for me to configure my signing info and sign with my key. For snapshots staying on my laptop, it shouldn't harm. I can do that as well.